### ISSUE TITLE
JSON generated from One Of is broken in Refract

### ISSUE THREAD

**Issue by honzajavorek** [apiaryio/drafter#288]
Hello gentlemen, Dredd user reports problems with One Of in https://github.com/apiaryio/dredd/issues/439. Sample blueprint:

```apib
# GET /
+ Response 200 (application/json)
    + Attributes (jsonTaskBase)

# Data Structures
## jsonTaskBase (object)
+ recurrence (jsonRecurrence)

## jsonRecurrence (object)
+ One of
    + numberOfOccurences: 5 (number)
    + endDate: `2015-04-08T12:30:00Z` (string)
+ One of
    + month: `march` (string)
    + interval: 1 (number)
```

Output:

```json
{
  "recurrence": {
    "": [
      [
        "march"
      ],
      [
        1
      ]
    ]
  }
}
```

Expected output:

```json
{
  "recurrence": {
    "numberOfOccurences": 5,
    "month": "march"
  }
}
```

**Comment by klokane** [comment]
seems to be #200 related

**Comment by pksunkara** [comment]
Fixed by #307
